controllers/tektontasks_controller.go

```diff
@@ -50,6 +50,7 @@ import (
 const (
 	finalizerName    = "tekton-tasks.kubevirt.io/finalizer"
 	oldFinalizerName = "finalize.tekton-tasks.kubevirt.io"
+	errMsg           = "there are multiple CRs deployed"
```

Comment:
If defining it here you should give it a more meaningful name. Moving the definition to the start of `setStatusMultipleCRs` would also be a good option.